merged_schema/HPXMLMerged.xsd

```diff
@@ -2532,189 +2532,289 @@
+						<xs:element minOccurs="0" name="Load">
 							<xs:complexType>
 								<xs:sequence>
-									<xs:group ref="SystemInfo"/>
-									<xs:element minOccurs="0" ref="ConnectedDevice"/>
-									<xs:element minOccurs="0" name="Type" type="PoolCleanerType">
-										<xs:annotation>
-											<xs:documentation>Type of pool cleaner used, if any.</xs:documentation>
-										</xs:annotation>
-									</xs:element>
-									<xs:element minOccurs="0" name="HoursPerDay" type="HoursPerDay">
-										<xs:annotation>
-											<xs:documentation>Hours per day pool cleaner is used. </xs:documentation>
-										</xs:annotation>
-									</xs:element>
-									<xs:element minOccurs="0" ref="extension"/>
+									<xs:element name="Units" type="PoolHeaterUnits"/>
+									<xs:element name="Value" type="HPXMLDouble"/>
 								</xs:sequence>
-								<xs:attribute name="dataSource" type="DataSource"/>
 							</xs:complexType>
 						</xs:element>
-						<xs:element minOccurs="0" name="Heater">
+						<xs:element minOccurs="0" ref="extension"/>
+					</xs:sequence>
+					<xs:attribute name="dataSource" type="DataSource"/>
+				</xs:complexType>
+			</xs:element>
+		</xs:sequence>
+		<xs:attribute name="dataSource" type="DataSource"/>
+	</xs:complexType>
+	<xs:complexType name="Pools">
+		<xs:sequence>
+			<xs:element maxOccurs="unbounded" name="Pool">
+				<xs:complexType>
+					<xs:complexContent>
+						<xs:extension base="PoolSpaType">
+							<xs:sequence>
+								<xs:element ref="extension"/>
+							</xs:sequence>
+						</xs:extension>
+					</xs:complexContent>
+				</xs:complexType>
+			</xs:element>
+		</xs:sequence>
+		<xs:attribute name="dataSource" type="DataSource"/>
+	</xs:complexType>
+	<xs:complexType name="StationarySpas">
+		<xs:sequence>
+			<xs:element maxOccurs="unbounded" name="StationarySpa">
+				<xs:complexType>
+					<xs:complexContent>
+						<xs:extension base="PoolSpaType">
+							<xs:sequence>
+								<xs:element ref="extension"/>
+							</xs:sequence>
+						</xs:extension>
+					</xs:complexContent>
+				</xs:complexType>
+			</xs:element>
+		</xs:sequence>
+		<xs:attribute name="dataSource" type="DataSource"/>
+	</xs:complexType>
+	<xs:complexType name="PortableSpas">
+		<xs:sequence>
+			<xs:element maxOccurs="unbounded" name="PortableSpa">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:group ref="SystemInfo"/>
+						<xs:element minOccurs="0" name="Manufacturer" type="HPXMLString"/>
+						<xs:element minOccurs="0" name="Model" type="HPXMLString"/>
+						<xs:element minOccurs="0" name="Capacity" type="IntegerGreaterThanZero">
+							<xs:annotation>
+								<xs:documentation>Number of people</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element minOccurs="0" name="Fuel" type="FuelType"/>
+						<xs:element minOccurs="0" name="Volume" type="Volume">
+							<xs:annotation>
+								<xs:documentation>[gal] Rated volume of hot tub.</xs:documentation>
```

Comment:
All the elements under `PortableSpa` refer to "hot tub", should they be changed to "portable spa or hot tub"?